### ISSUE TITLE
ImageView tintColor not working

### ISSUE THREAD

**Issue by krishnameena** [rs/SDWebImage#2090]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

When I am loading  static image via `imageView.sd_setImage(with: nil, placeholderImage: image)` imageview tintColor is not working.

**Comment by bpoplauschi** [comment]
@krishnameena please fill in the info required in the issue template.
Also, why are you loading a nil url? That makes no sense.
I'm pretty sure the tintColor has nothing to do with our library.

**Comment by krishnameena** [comment]
Hi, As I mentioned earlier that I am loading static image with sdwebimage library so I have set url to nil.

**Comment by krishnameena** [comment]
Hi, Apology for mistake, It is some other issue and I got it resolved.

**Comment by bpoplauschi** [comment]
If you're loading a static image, just set it `imageView.image = image`, no need for the SDWebImage functions.

**Comment by krishnameena** [comment]
Thanks for the information.

**Comment by dimpy-iroid** [comment]
Hello @bpoplauschi, 
How can I change the tintColor of the ImageView?
this is my code:
```
UtilityMethodsImageVideoAudio.setImageForProfilePhoto(object?.icon, imageView: cell.privacyControlImageView)
cell.privacyControlImageView.tintColor = .red
```

but still my imageView's tintColor is not updating to red.
